pkg/cmd/codespace/create.go

```diff
@@ -466,12 +466,15 @@ func getRepoSuggestions(ctx context.Context, apiClient apiClient, partialSearch
 }
 
 // buildDisplayName returns display name to be used in the machine survey prompt.
+// prebuildAvailability will be migrated to use enum values: "none", "ready", "in_progress" before Prebuild GA
+// Enum values "blob" and "pool" will be deprecated soon.
 func buildDisplayName(displayName string, prebuildAvailability string) string {
-	prebuildText := ""
-
-	if prebuildAvailability == "blob" || prebuildAvailability == "pool" {
-		prebuildText = " (Prebuild ready)"
+	switch prebuildAvailability { 
+		case "blob", "pool", "ready": 
```

Comment:
Note that the Go linter is failing because `go fmt` needs to be run on this file to correct the indentation. For future contributions, it might be a good idea to configure the text editor to automatically format Go files on save—that helps me since otherwise I would forget to do it.

Reply:
Thank you! I will update.